### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by booee**
Documentation on how to install tool + local source... and verify that installation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
+$ hewer -v
+```
+
+### Usage
+
+```
+$ hewer <fileName> -k <keyName>
+```
+
+run `$ hewer help` for more information
```
Comment (nexdrew) [inline README.md]: This should be `hewer --help`. Otherwise, it thinks you want to parse a file called "help".

File: README.md [hunk 2]
```diff
@@ -1,7 +1,28 @@
 # hewer
+v0.1.0
 
-### Need
+#### Installation Prerequisites
+1. Have go-lang installed ([duh](http://lmgtfy.com/?q=go-lang+install+download))
```
Comment (nexdrew) [inline README.md]: Nice use of lmgtfy

File: README.md (hunk 1 above)
Comment (booee) [inline README.md]: Nice catch. I also noticed some urls weren't ideal, either. Fixed

**Comment by booee** [comment]
Added a few more tweaks to the readme based on the fact that some ppl may actually need to use it.

**Comment by nexdrew** [comment]
:+1: Approved. Merge at will.

**Pull request merged**